src/include/rbtree.h

```diff
@@ -1,25 +1,58 @@
 /*
- * The primary function of red-black trees in Box64 is to provide a fast and efficient method (O(log(n))) for managing memory mappings. 
- * Each red-black tree node includes two additional pointers, "start" and "end," to denote a specific memory range. 
- * When a range of memory is mapped from the system, it is recorded in red-black trees based on the type and characteristics of that memory range.
+ * The primary function of red-black trees in Box64 is to provide an O(log(n)) method for managing memory mappings.
```

Comment:
Change `O(log(n))` to `O(log n)` as the standard mathematical notation for Big O complexity does not use parentheses around the variable.